repos/c2sm/packages/icon/package.py

```diff
@@ -640,9 +640,16 @@ def configure_args(self):
                     'liskov does not support fusing just yet')
 
             config_vars['LOC_GT4PY'].append(self.spec['py-gt4py'].prefix)
-            config_vars['LOC_ICON4PY'].append(
-                os.path.join(self.spec['py-icon4py'].prefix))
-            config_vars['LOC_ICON4PY_LIB'].append(
+            config_vars['LOC_ICON4PY_BIN'].append(
```

Comment:
This solution is a bit dirty, because it exposes some details of the icon4py-installation in another package.
Better is to use a predefined interface for this. Spack offers the `libs` function , see https://github.com/C2SM/spack-c2sm/blob/56ed256ba2e6bdab06323482bc4ece77a91d5199/repos/c2sm/packages/infero/package.py#L72 for an implementation example.
Instead of a library-list this function would return the path.

Reply:
yeah this makes sense. I'll have a look at it after my break.